javascript/test/CucumberExpressionGeneratorTest.ts

```diff
@@ -63,37 +64,110 @@ describe('CucumberExpressionGenerator', () => {
   })
 
   it('generates expression with escaped slashes', () => {
-    assertExpression('The {int}\\/{int}\\/{int} hey', ['int', 'int2', 'int3'], 'The 1814/05/17 hey')
+    assertExpression(
+      'The {int}\\/{int}\\/{int} hey',
+      [
+        {
+          type: 'Number',
+          name: 'int',
+          nameSuffix: '',
```

Comment:
This is effectively using the empty string to mean `undefined`.

Reply:
Fixed!